projects/cli-files/go-cat/cmd/root.go

```diff
@@ -0,0 +1,69 @@
+package cmd
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"os"
+)
+
+func Execute() error {
+
+	numberLine := flag.Bool("n", false, "Add number in front of each line")
+	help := flag.Bool("h", false, "Show help")
+	helpFilePath := "assets/help.txt"
+
+	flag.Parse()
+
+	var pathArr []string
+
+	if *numberLine && *help {
+		pathArr = os.Args[3:]
+	} else if !*numberLine && !*help {
+		pathArr = os.Args[1:]
+	} else {
+		pathArr = os.Args[2:]
+	}
+
+	if *help {
+		err := OpenFile(helpFilePath, false)
```

Comment:
This works here, but it relies on there being a file named `assets/help.txt` relative to the working directory wherever this program is run. If you `cd` to `/tmp` and try to `go-cat -h` it will fail, or if you just upload `go-cat` somewhere and someone downloads it, `go-cat -h` will fail. Can you think how to avoid this problem?